test/e2e/specs/fakerp/cvepatch.go

```diff
@@ -0,0 +1,112 @@
+package fakerp
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/ghodss/yaml"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/pkg/fakerp"
+	"github.com/openshift/openshift-azure/pkg/util/azureclient"
+	"github.com/openshift/openshift-azure/test/clients/azure"
+	"github.com/openshift/openshift-azure/test/sanity"
+	logger "github.com/openshift/openshift-azure/test/util/log"
+)
+
+var _ = Describe("Apply CVE hot patch E2E tests [ApplyHotPatch][Fake][LongRunning]", func() {
+	var (
+		ctx       = context.Background()
+		patchRpms = []string{
```

Comment:
this list of rpms works for the checks in this test however I feel the test will break if this is run at a later date against a cluster with VMs which have been built with those rpms already in place.